tests/checkings.test.js

```diff
@@ -0,0 +1,96 @@
+import HomePage from "../pages/HomePage";
+import SignIn from "../pages/SignIn";
+import CheckingPage from "../pages/CheckingPage"
+ 
+
+describe('Checkings', () => {
+it('Should take you to Checkings form', async ()=> {
+    await SignIn.open('bank')
+    let username = 'jsmith@demo.io'
+    let password = 'Demo123!'
+    await SignIn.logIn(username, password)
+    await HomePage.displayCheckingMenu() 
+    await HomePage.accessNewCheckingForm()
+    await expect(HomePage.checkingTitle).toHaveText('Create Checking')
+})
+it('Should take you to Checkings View', async ()=> {
+    await HomePage.displayCheckingMenu() 
+    await HomePage.accessViewCheckingForm()
+    await expect(HomePage.checkingTitle).toHaveText('View Checking Accounts')
+})
+it('Should see the three pages going forwards and backwards', async ()=> {
+    await CheckingPage.changeCheckingFormPageNext()
+    await expect(CheckingPage.checkingTableTextInfo).toHaveText('Showing 11 to 20 of 24 entries')
+    await CheckingPage.changeCheckingFormPageNext()
+    await expect(CheckingPage.checkingTableTextInfo).toHaveText('Showing 21 to 24 of 24 entries')
+    await CheckingPage.changeCheckingFormPagePrevious()
+    await expect(CheckingPage.checkingTableTextInfo).toHaveText('Showing 11 to 20 of 24 entries')
+    await CheckingPage.changeCheckingFormPagePrevious()
+    await expect(CheckingPage.checkingTableTextInfo).toHaveText('Showing 1 to 10 of 24 entries')
+})
+it('Should filter the table with the word "Income"', async ()=> {
+    await CheckingPage.sendText(CheckingPage.checkingSearchInput, 'Income')
+    await expect(CheckingPage.checkingTableTextInfo).toHaveTextContaining('(filtered from 24 total entries)')
```

Comment:
IMO, in order to assert this scenario, you have to make sure no other entry but the ones that have text "Income" are display in the table after the search. Therefore you need to implement a process that checks every row and validates that all of them (in the relevant column) displays "Income". If one of them don't, then the test fails.

Implement the assertion considering that logic

Reply:
By using a class, does the test checks every class or only the first one?